packages/react/react/src/components/Portal/Portal.tsx

```diff
@@ -0,0 +1,22 @@
+import ReactDOM from 'react-dom';
+import { useIsMounted } from '../../hooks/useIsMounted';
+
+interface Props {
+  children: React.ReactNode;
+  id: string;
```

Comment:
How about trying to find the parent's node to render using the `ref` value instead of the `id`(string)?

Looking at the various implemented libraries, it seems to be implemented as `ref`, not as the string value id.

If it is implemented as an `id`, which is a normal magic string, it is difficult to find which file has the element with the `that id`, and it is likely to cause side effects.

However, if you manage it with `ref`, the location of the parent node can be directly identified on the code, so I think the developer experience will be good 

ref

[chakra-ui](https://chakra-ui.com/docs/components/portal/props)
[mui](https://mui.com/base-ui/react-portal/components-api/)

Reply:
Thanks for the great feedback. 😀

Definitely, managing by `id(string)`  is more likely to cause side effects, and it's a bad developer experience. 

I will think about a Portal Component managed by `ref`. 

Additionally, the default behavior of "mui" is to add a span tag to the "body" tag, and "chakra ui" is to add a div tag with "chakra-portal" as class to the body tag.

The default behavior of chakra ui seems more appropriate to me, and I will try to design similarly to it.

Reply:
@ChanhyukPark-Tech 
Oh sorry, I was mistaken about the MUI default behavior.